### ISSUE TITLE
Force Recalc after API Post?

### ISSUE THREAD

**Issue by ZohraS** [audreyt/ethercalc#385]
Hi guys,

New problem here. I have an outside form adding and retrieving data from an ethercalc multisheet. 

Page 2 of the sheet has a sumif, and it should get recalculated every time we post some info in Page 3. For some reason, it starts ignoring completely the column with the formula. Even fields that don't need the recalc go sort of stupid, and just display a completely false value.
I've been checking out the api, but what I wanted to know is this:
Is there a way to force the sheets to recalculate after adding data via POST? 

Thanks!

**Comment by audreyt** [comment]
Yes. POST `recalc` will force a recalc. http://docs.ethercalc.apiary.io/#reference/0/page/post-commands

**Comment by ZohraS** [comment]
Hey Audrey,

I tried that in every possible way, and get nothing.
I'm  using a multi-sheet, so I tried posting to http://example.com/_/nameofmulti.2 Then tried http://example.com/_/nameofmulti . Tried changing from public url to ip and localhost, changed it from recalc to ask.recalc. Tried it via node, and via javascript. No matter what I do, it doesn't go through. Either I get  weird PNR problem, or the page loads forever, or ethercalc crashes. 
Any ideas?

**Comment by audreyt** [comment]
Would it be possible to post a link to your Redis RDB (or preferably the `dump.json` by running ethercalc with environment `REDIS_PORT=9999` to force json storage) so we have a chance to reproduce?

**Comment by ZohraS** [comment]
If you gimme your email addy, I'll send it to you.

**Comment by audreyt** [comment]
audreyt at audreyt.org (I'll look into it sometime this weekend but if you post a reduced sample somewhere public other folks here may be helpful sooner)
